Makefile

```diff
@@ -94,11 +95,11 @@ generate: controller-gen
 
 # Build the docker image
 docker-build: test
-	docker build -t ${IMG} .
+	docker buildx build --tag ${IMG} --platform ${image_arch_list} .
 
 # Push the docker image
-docker-push:
-	docker push ${IMG}
+docker-build-and-push:
```

Comment:
rename it to `dockerx-build-and-push-multi-arch`

Reply:
done